apps/api/src/server/repositories/case.repository.js

```diff
@@ -439,6 +441,8 @@ export const updateApplicationStatusAndDataById = (
 		transactions.push(assignApplicationReference(id));
 	}
 
+	transactions.push(createFolders);
```

Comment:
How does `push` work? If `createFolders` is an array, is it merged into `transactions` as a 1D array?